modules/fhir-client/test/blaze/fhir_client_test.clj

```diff
@@ -312,3 +312,25 @@
         ::anom/category := ::anom/fault))))
 
 
+(deftest basic-auth-client
+  (testing "Authenticator is set"
+           (let [username "user-134952"
+                 password "password-135000"
+                 client (fhir-client/authenticated-client "http://localhost:8080/fhir" username password)
+                 optionalAuthenticator (-> client
+                                           :http-client
+                                           .authenticator)]
+             (is (.isPresent optionalAuthenticator)))
+           )
```

Comment:
this parenthesis should not be on its own line

Reply:
Fixed